panel/config.py

```diff
@@ -394,7 +398,7 @@ def __setattr__(self, attr, value):
         if not init or (attr.startswith('_') and attr.endswith('_')) or attr == '_validating':
             return super().__setattr__(attr, value)
         value = getattr(self, f'_{attr}_hook', lambda x: x)(value)
-        if attr in self._globals or self.param._TRIGGER:
```

Comment:
This is a major speed impediment because

- It recursively looks up something that is a constant in my understanding.
- Later we run `__contains__` which is slow because the value is a list.